src/modules/IsShopgateApp.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Determines if the current browser is a Shopgate App or not
+ *
+ * @returns {*}
+ */
+export function isShopgateApp() {
```

Comment:
The general check if we have a Shopgate here or not is checking for window.SGJavascriptBridge. Therefore I would name it as it is - I already had the same thought but because iOS is injecting the bridge in specific scenarios later we need special logic.